docs/FAQs.md

```diff
@@ -0,0 +1,146 @@
+# FAQ'S
+Contents of this document.
+* [General](#general)
+* [Deployment and config](#deployment-and-config)
+* [API](#api)
+* [Handler](#handler)
+* [Model-archiver](#model-archiver)
+
+## General
+Relevant documents.
+- [Torchserve readme](https://github.com/pytorch/serve#torchserve)
+
+### Does Torchserve API's  follow some REST API standard?
+Torchserve API's are compliant with the [OpenAPI specification 3.0](https://swagger.io/specification/).
+
+### What is not Torchserve?
+Torchserve is not a complete web application to serve end to end business use cases. Hence a lot of security aspects should be made available via 3rd party wrapper components in front of Torchserve.
+
+### What's difference between Torchserve and a python web app using web frameworks like Flask, Django?
+The Flask app and Torchserve are completely different except the fact that both support handling HTTP requests however using different engines [netty or python apis].
+
+###  Are there any sample Models available?
+Various models are provided in Torchserve out of the box. Checkout out Torchserve [Model Zoo](https://github.com/pytorch/serve/blob/master/docs/model_zoo.md) for list of all available models in model zoo. Also check out examples folder all available [examples](https://github.com/pytorch/serve/tree/master/examples).
+
+### Does Torchserve has Windows Support?
+Currently, Torchserve supports Windows only via WSL(Windows Subsystem for Linux).
+The native Windows support will be added in upcoming releases.
+
+### Can I do streaming service with Torchserve like streaming speech recognition?
+Torchserve currently supports only inference through HTTP 1.0 - Request / Response style.
+We don't have anything particular in our roadmap - [pytorch/pytorch#27610](https://github.com/pytorch/pytorch/issues/27610)
+
+### Is it possible to deploy model other than Pytorch framework?
+Yes, it is possible to deploy(with Python inference logic) however Torchserve has been certified with any other framework hence there can be unknowns.
```

Comment:
Can you please add comments to issues / specific customer ask for this question. Seems a bit too general